server/s3_bucket_metadata.cc

```diff
@@ -279,6 +279,32 @@ std::string S3BucketMetadata::get_tags_as_xml() {
   return tags_as_xml_str;
 }
 
+std::string S3BucketMetadata::get_bucket_versioning_status_as_xml() {
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
+  std::string versioning_status;
+  std::string versioning_status_as_xml_str;
+
+  if (bucket_versioning_status != "Unversioned") {
+    versioning_status +=
+        "<VersioningConfiguration "
+        "xmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\">" +
+        S3CommonUtilities::format_xml_string("Status",
+                                             bucket_versioning_status.c_str()) +
+        "</VersioningConfiguration>";
+
+  } else {
+    versioning_status +=
+        "<VersioningConfiguration "
+        "xmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\"/>";
```

Comment:
`s3.amazonaws.com/`
should this be
`s3.seagate.com/`

Reply:
It's correct as-is, we're implementing the AWS API and so we need to return the same responses. This indicates which version of the AWS S3 specification is being used. All S3-compatible servers use this same XML namespace.

Reply:
done

Reply:
Never mind, reverted the same.